docs/dataset/advanced-usage/extract-text-metadata.md

```diff
@@ -68,6 +68,7 @@ The following properties are available for automatic text metadata extraction:
 | [Misspelled Count](#misspelled-count) | Count of misspelled words |
 | [Named Entity Count](#named-entity-count) | Number of named entities in the text |
 | [Toxicity Flag](#toxicity-flag) | Flags potentially toxic content |
+| [Question Flag](#question-flag) | Identifies whether the text is a question |
```

Comment:
Since we are considering ordering the properties in alphabetical order, should we also apply the same ordering here?

Reply:
Reordered them (also on index.md/mkdocs.yml)